src/main/java/com/rocketden/main/controller/v1/UserSocketController.java

```diff
@@ -0,0 +1,19 @@
+package com.rocketden.main.controller.v1;
+
+import com.rocketden.main.model.User;
+
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserSocketController extends BaseRestController {
+
+    @MessageMapping("/user")
+    @SendTo(BaseRestController.BASE_SOCKET_URL + "/subscribe-user")
+    public User user(String nickname) {
+        User user = new User();
+        user.setNickname(nickname);
+    	return user;
+    }
```

Comment:
Optional nit: Fix indent, and create a DTO/response/request class(es) for User. That comes with a lot of boilerplate though (extra classes, mappers, service class, tests) so I think we can also hold off on this until it gets a bit more complex.

Reply:
I fixed the indentation, thanks for the note. 👍

As for creating the DTO/response/request classes for the User, I think I'll add that on future PRs, as it's not as necessary for this one IMO.